expandybird/expander/expander_test.go

```diff
@@ -50,23 +57,152 @@ func (etc *ExpanderTestCase) GetTemplate(t *testing.T) *common.Template {
 func GetOutputString(t *testing.T, description string) string {
 	output, err := ioutil.ReadFile(outputFileName)
 	if err != nil {
-		t.Errorf("cannot read output file for test case '%s': %s\n", description, err)
+		t.Fatalf("cannot read output file for test case '%s': %s", description, err)
 	}
 
 	return string(output)
 }
 
+func expandOutput(t *testing.T, actualOutput, description string) {
```

Comment:
seems like this should be named:
expandOutputAndVerify or something cause it's not only expanding, but it is also validating?